### ISSUE TITLE
Screen Reader: Many Pages: Emoji category buttons announced with variables

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77424]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack on Android, open the app
2. Swipe right to focus the Account tab and double-tap to activate it
3. Swipe right to focus the Profile button and double-tap to activate it
4. Swipe right to focus the Status menu item and double-tap to activate it
5. Swipe right to focus the Status Emoji button and double-tap to activate it
6. Swipe right to focus the emoji category emojis and note how they are announced

This issue reproduces in all screens with the emoji option
Examples:
1. New chat
2. Group chat

## Expected Result:
Screen readers should announce a clear and concise category name, e.g.:
"Frequently used"
"Smileys,
"Animals"

## Actual Result:
When opening the Status emoji selection dialog, the grey emoji category buttons at the top (e.g., frequently used, smileys, activities, etc.) are announced with internal keys, for example:
"emoji Picker.headers.frequently used, button"

This is a poor accessible label and exposes internal keys.

## Area issue was found in:
Preferences and  Subsections

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Provide a clean, user-facing accessibilityLabel to all emoji category buttons, for example:
"Frequently used"
"Smileys"
"Animals".
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment(s)
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
<img width="1179" height="2428" alt="Image" src="https://github.com/user-attachments/assets/48b5509a-3bc2-4cea-b63f-2bca6c507b4c" />

https://github.com/user-attachments/assets/02fddba3-696e-47c3-aabc-35661c914988

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7008032
**Issue reported by:** NA
**Slack conversation:** NA
